packages/ml/src/Utils/ml_MultiLevelPreconditioner.cpp

```diff
@@ -2152,6 +2152,15 @@ ComputePreconditioner(const bool CheckPreconditioner)
       ml_->Amat[LevelID_[0]].num_PDEs = NumPDEEqns_;
     }
 
+    if (List_.get("aggregation: material: enable", false)) {
```

Comment:
Woah, what's this?  Should we still have this here?  Do we need to rebase again or otherwise account for the latest changes in develop?

Reply:
Argh, I messed up the merge. But now, a recursive diff of the ml directories in this branch and develop shows no changes, which is how it should be.